data-description/fair-data.md

```diff
@@ -0,0 +1,128 @@
+* Check the accessibility of your data. Your dataset, when submitted to an archive, is likely to have a landing page with several data access services (e.g., download, transcription, contact person, visualisation…). From the perspective of someone foreign to the system to access the data, try to find out how easy it is and then adjust and retest. To increase the quality of accessibility, you can also apply the stable [W3C WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) and use their tools and methods to assess it.
+
+### Existing approaches 
+
+* There are multiple archives with secure procedures already in place for the distribution of sensitive human information through authentication and granted access. For example, the {% tool "ega" %} has a request and grant method to provide secure ad-hoc access to human datasets.
+* Check other use-cases and examples at the [Data access]((/human-clinical-and-health-data/data-sources)) section.
+
+
+## Interoperability
+
+Interoperability is essential for infectious diseases research because it enables the **integration and analysis of data from different sources** (e.g. different hospitals, countries, biological sources, etc.), leading to a more comprehensive understanding of disease transmission, prevention, and treatment. 
+
+Without interoperability, data silos may emerge, restricting researchers' ability to link different datasets and slowing progress in the fight against infectious diseases. Specifically, issues such as synonymy and polysemy can lead to misinterpretation of scientific results and hinder the clear communication of recommendations, which can be crucial, especially during a pandemic.
+
+### Considerations
+
+* Provide detailed metadata for infectious disease datasets, including the source, collection date, location, and any performed protocols (e.g. nasal swab being the method of isolation: [EFO:0010741](http://www.ebi.ac.uk/efo/EFO_0010741)). Even when the granularity of the (meta)data varies, you should always use descriptive fields with broadly understandable values. 
+* Use controlled vocabularies and ontologies to describe human data and infectious diseases (e.g. [EFO:0007182](http://www.ebi.ac.uk/efo/EFO_0007182) for Brill-Zinsser disease). Furthermore, do not forget contextual data that must meet intercommunity standards, for example: time, temperature, pressure, chemical components… 
+* Controlled vocabulary refers to a set of terms, standardised by the field community,  used to describe and categorise concepts, ensuring consistency and accuracy in data organisation and retrieval. For example, when a disease (e.g. Alport syndrome) has multiple used names (e.g. Alport deafness-nephropathy), it is recommended to use the designated one in the ontologies, so the redundancy is kept to a minimum.
```

Comment:
I think Alport syndrome is not an infectious disease. Then I'd either replace it by an example of a infectious disease or drop it.